feature/client/src/main/java/com/mifos/feature/client/navigation/ClientNavigation.kt

```diff
@@ -59,31 +67,31 @@ fun NavGraphBuilder.clientNavGraph(
             uploadSignature = navController::navigateClientSignatureScreen,
             loanAccountSelected = loanAccountSelected,
             savingsAccountSelected = savingsAccountSelected,
-            activateClient = activateClient
+            activateClient = activateClient,
         )
         clientChargesRoute(
-            onBackPressed = navController::popBackStack
+            onBackPressed = navController::popBackStack,
         )
         clientIdentifierRoute(
             onDocumentClicked = {},
-            onBackPressed = navController::popBackStack
+            onBackPressed = navController::popBackStack,
         )
         clientPinPointRoute(
-            onBackPressed = navController::popBackStack
+            onBackPressed = navController::popBackStack,
         )
         clientSignatureRoute(
-            onBackPressed = navController::popBackStack
+            onBackPressed = navController::popBackStack,
         )
         clientSurveyListRoute(
             onBackPressed = navController::popBackStack,
-            onCardClicked = { _, _ -> }
+            onCardClicked = { _, _ -> },
         )
         clientSurveyQuestionRoute(
-            onBackPressed = navController::popBackStack
+            onBackPressed = navController::popBackStack,
         )
         createClientRoute(
             onBackPressed = navController::popBackStack,
-            hasDatatables = { _, _ -> }
+            hasDatatables = { _, _ -> },
```

Comment:
fix missing routes here.
![image](https://github.com/user-attachments/assets/84d5c8df-be8a-4118-92f5-41ebf432ea2d)

Reply:
On it! everything has been fixed, apart from onCardClick. There seems to be an issue there.

Reply:
about that, let's connect tonight if you are free